.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,12 +1,12 @@
 blank_issues_enabled: false
 contact_links:
-  - name: "💬 Jupyter community Discourse"
+  - name: "💬 - Jupyter community Discourse"
     url: https://discourse.jupyter.org/
     about: Interact with the rest of the Jupyter community
-  - name: "📖 Jupyter Docker Stacks documentation"
+  - name: "📖 - Jupyter Docker Stacks documentation"
```

Comment:
I think the documentation should go first.

Reply:
💯  agree